us-enrichment-api/client.go

```diff
@@ -36,6 +36,12 @@ func (c *Client) SendPropertyPrincipal(lookup *Lookup) (error, []*PrincipalRespo
 	return err, propertyLookup.Response
 }
 
+func (c *Client) SendPropertyGeoReference(lookup *Lookup) (error, []*GeoReferenceResponse) {
+	geoReferenceLookup := &geoReferenceLookup{Lookup: lookup}
```

Comment:
This variable name `geoReferenceLookup` is overriding the struct with that same name. Functionality will be preserved but I recommend changing the variable name just for the sake of cleaner code and to prevent bugs in the future.